cdk/lib/constructs/api.ts

```diff
@@ -44,6 +67,21 @@ export class BackendApi extends cdk.Construct {
 
     this.addPrivateEndpoints(apiIntegration, authorizer);
     this.addPublicEndpoints(publicApiIntegration);
+
+    const key = this.api.addApiKey("PerformanceDashboardApiKey");
+    const plan = this.api.addUsagePlan("PerformanceDashboardUsagePlan", {
+      name: "PerformanceDashboardUsagePlan",
+      apiKey: key,
+      throttle: {
+        rateLimit: 100,
+        burstLimit: 200,
```

Comment:
I suggest we go a little bit more conservative here. 100/200 requests per second is a lot considering that an AWS account has 1,000 concurrent lambda executions by default and considering that the Ingest API has to load data into S3 so the latency is not as good as it is for other endpoints. 

My suggestion is: 
```
rateLimit: 25
burstLimit: 50
```